packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -0,0 +1,40 @@
+// Port of Sindre Sorhus' Sparkly to Rescript
+// reference implementation: https://github.com/sindresorhus/sparkly
+// Omitting rgb "fire" style, so no `chalk` dependency
+
+type sparklyConfig = {
+  minimum: option<float>,
+  maximum: option<float>
+}
+
+let sparkly = (
+  numbers: array<float>,
+  ~options = {minimum: None, maximum: None}
+) => {
+  // Unlike reference impl, we assume that all numbers are finite, i.e. no NaN.
+
+  let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
+
+  let minimum = E.O.default(Js.Math.minMany_float(numbers), options.minimum)
+  let maximum = E.O.default(Js.Math.maxMany_float(numbers), options.maximum)
+
+  // Use a high tick if data is constant and max is not equal
+  let ticks = if minimum == maximum && maximum != 0.0 {
```

Comment:
I take it this doesn't support negative numbers? Maybe have a check for this.

Reply:
You can pass in negative numbers, you just won't know where the x intercept is.

Reply:
@OAGr would you like a check and reject negative numbers?